### DISCUSSION TITLE
Bug des coffre

### DISCUSSION THREAD

**Discussion by EYESMETAL** [RandyKnapp/Auga#21] [Q&A]
Petit souci ennuyeux quand j’ouvre mes coffre je ne vois pas le contenu?

**Comment by nullspeaker** [comment]
Vous pourrez peut-être contourner ce problème en passant en mode fenêtré et en jouant avec la résolution.

Mon ami avec un écran au format large voit ce problème et est capable de voir le widget de l'interface utilisateur de la poitrine lorsqu'il modifie la taille de la fenêtre.

(Google traduit, désolé pour la mauvaise formulation !)

Éditer:
Peut être lié au #65 si vous avez "Epic Valheim" 
Et comme l'indique #65, cela pourrait provenir de tout autre mod qui modifie l'interface utilisateur mais ne prend pas en charge [l'API Auga](https://github.com/RandyKnapp/Auga/tree/main/AugaApiExample).
